cmd/podman/main_local.go

```diff
@@ -153,3 +155,12 @@ func setupRootless(cmd *cobra.Command, args []string) error {
 	}
 	return nil
 }
+func startTrace() {
+	if MainGlobalOpts.Trace {
```

Comment:
When is this parsed? I feel like it'd be better to pull it out of cobra.Command instead of using a global field